LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2008 Alain Frisch
+Copyright (c) 2008 - 2019 Alain Frisch and 2019 - opto (Klaus Buecher)
```

Comment:
Since the new version has evolved from the previous one and still contains significant portions of it, could you keep me as a copyright holder (in addition to you, post 2019)?